### DISCUSSION TITLE
gitlab_url Usage

### DISCUSSION THREAD

**Discussion by carlosecv** [Tecnativa/doodba#357] [Doodba - The template]
This can be a very basic questions but i didnt figure it out how to doit correctly:

When we create a copier template it asks for gitlab_url, ¿what is the correct use of this gitlab_url?:

¿It refers to gitlab repository created to get all this configuration directory on it?
¿Do I have to create previously this repo?
¿This repo its an empty repo or  we have to clone some repo previously?

¿Someone have some steps to doit correctly?

Regards

**Comment by yajo** [comment]
The question help answers most questions: https://github.com/Tecnativa/doodba-copier-template/blob/0365f652e79c8dd1a6e89720172371ddfb5558bb/copier.yml#L190-L196

We don't do anything fancy with that. It's only used to display some badges in the README. It refers to the repo where your doodba subproject will live. We don't create it for you.
